va/va.h

```diff
@@ -129,6 +129,98 @@ extern "C" {
  * 	- \ref api_fei
  * 	- \ref api_fei_h264
  * 	- \ref api_fei_hevc
+ * 
+ * \section threading Multithreading Guide
+ * All VAAPI functions implemented in libva should be thread-safe, and the 
```

Comment:
Please be careful with the wording. I'd recommend sticking to what https://www.ietf.org/rfc/rfc2119.txt suggests, and in particular the use of MUST and SHOULD. That said, this paragraph is misleading, because it starts saying "A is guaranteed" then "B should be guaranteed" but then says that if "B is not verified" then "A is not verified either".

Recommended writing:
> All VAAPI functions implemented in libva are thread-safe insofar as the driver and backend
> implementation are thread-safe. Note that libva cannot impose thread-safety guarantees
> on neither the driver nor the backend, so it is ultimately the responsability of the latter to
> ensure said thread-safety.

Please fix "`thread-saftey`" with "`thread-safety`" (ideally also introduce a ref-link
to e.g. https://en.wikipedia.org/wiki/Thread_safety).

Reply:
Thanks for the suggested changes and it's been incorporated into the latest version